GoogleForCreators/web-stories-wp
# Floating Menus: Add Font Family

## Context



For use in the floating menus we want to refactor the existing font family dropdown so that we can use the same dropdown in both places. The width / height varies between the two use cases. 

## Summary



We split fontPicker and storyFontPicker into two components in #10848 Here we really jut needed to add some styling props since the use case in the side panel and in the floating menu is slightly different.

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. open floating menu by turning on experiment and selecting text element
2. check that you can change font and it stays in sync with side panel. 


## Reviews

### Does this PR have a security-related impact?
no


### Does this PR change what data or activity we track or use?
no


### Does this PR have a legal-related impact?
no


## Checklist




---



Fixes #10628
